### DISCUSSION TITLE
Karma toggle in the main menu is confusing

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#13147] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The title "Playing with friends?" does not tell you that it disables karma unless you hover over it and read the tooltip. It also words karma as something that should be left enabled most of the time, which is not true.
![image](https://github.com/Regalis11/Barotrauma/assets/86113712/2bfdfb2d-04ea-4893-9c50-15b2cfca7bb8)

### Reproduction steps

Open the game.
Click on "Host server"
Notice that it tells you nowhere besides hovering over the button that "Playing with friends?" disables karma, and if you DO hover over it, the wording is incredibly favourable towards the system.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.1.19.3 (Treacherous Tides Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
Not to mention, in terms of it's actual intended purpose as some kind of automod it does not perform very well. It is very easy to exploit by any determined griefer (by indirect sabotage and tricking the system to cause other people to lose karma other than them) while simultaneously getting in the way of people trying to play the game organically; either by friendly fire, medical treatment, or bizarre ship-damage calculations. 

I notice alot of crews with 'karma' enabled tend to play very passively and avoid personal agency so as to not step on the toes of the karma system, to give a specific example; being less aggressive in responding to threats on the ship for fear of accidental friendly fire, or being unable to use grenades on ballast flora (a well known trick by pretty much everybody) due to it 'damaging the ship'

> It also words karma as something that should be left enabled most of the time, which is not true. 

> the wording is incredibly favourable towards the system.

That is a polite way of putting it yes, 
and it starts enabled by default - luckily alot of people have had realized to turn it off if they don't want it.
